compiler/src/iree/compiler/Dialect/Flow/IR/FlowOps.td

```diff
@@ -38,11 +39,20 @@ def FLOW_DispatchRegionOp : FLOW_PureOp<"dispatch.region", [
     isolated from above. `dispatch.region` ops are a more lightweight
     abstraction for implementing fusion heuristics, i.e., the process of
     deciding which ops should form a dispatch region.
+
+    This op also has a second region: `workload_count`. This region calculates 
```

Comment:
Nit : Second sentence could be "The arguments to the region represent the workload for the dispatch, and returns the number of workgroups for the dispatch."